From 5a107de9cc80b5244b1f888bb5d25d458a6a9c8b Mon Sep 17 00:00:00 2001
From: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
Date: Wed, 22 May 2013 22:57:08 +0200
Subject: [PATCH 297/509] dma: xilinx: vdma: Turn channel flush_fsync field
 into a bool

https://github.com/analogdevicesinc/linux.git xcomm_zynq
commit 0cfe1f49a5f367c9e18120d47e3c1ec7204813ce

The field stores a boolean, make it bool.

Signed-off-by: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
Acked-by: Srikanth Thokala <sthokal@xilinx.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 drivers/dma/xilinx/xilinx_axivdma.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/dma/xilinx/xilinx_axivdma.c b/drivers/dma/xilinx/xilinx_axivdma.c
index 2dc6ff5..0efed82 100644
--- a/drivers/dma/xilinx/xilinx_axivdma.c
+++ b/drivers/dma/xilinx/xilinx_axivdma.c
@@ -174,7 +174,7 @@ struct xilinx_vdma_chan {
 							channel request */
 	void (*start_transfer)(struct xilinx_vdma_chan *chan);
 	struct xilinx_vdma_config config;	/* Device configuration info */
-	u32 flush_fsync;			/* Flush on Fsync */
+	bool flush_fsync;			/* Flush on Fsync */
 };
 
 struct xilinx_vdma_device {
@@ -1089,7 +1089,7 @@ static int xilinx_vdma_chan_probe(struct xilinx_vdma_device *xdev,
 
 		if (xdev->flush_fsync == XILINX_VDMA_FLUSH_BOTH ||
 		    xdev->flush_fsync == XILINX_VDMA_FLUSH_MM2S)
-			chan->flush_fsync = 1;
+			chan->flush_fsync = true;
 	} else if (of_device_is_compatible(node, "xlnx,axi-vdma-s2mm-channel")) {
 		chan->direction = DMA_DEV_TO_MEM;
 		chan->id = 1;
@@ -1099,7 +1099,7 @@ static int xilinx_vdma_chan_probe(struct xilinx_vdma_device *xdev,
 
 		if (xdev->flush_fsync == XILINX_VDMA_FLUSH_BOTH ||
 		    xdev->flush_fsync == XILINX_VDMA_FLUSH_S2MM)
-			chan->flush_fsync = 1;
+			chan->flush_fsync = true;
 	}
 
 	/*
-- 
1.7.5.4

